tests/remote_attestation/maa/sgxlkl_enclave/oe_public/bits/report.h

```diff
@@ -0,0 +1,207 @@
+/**
+ * @file report.h
```

Comment:
This file should go away if we expose the cert-related APIs through OE host verify.

Reply:
There is no action item for now. if this happens we can submit a follow up PR. Let's don't block the PR with uncertainity for now.

Reply:
This is the one of the first two design suggestions I gave. I feel strongly we shouldn't introduce so many oe_private / oe_public files for this test case.